build/build.py

```diff
@@ -67,8 +67,36 @@ def apply_patch(patch):
             print(test["package"], "-run", testName)
             print(go.test(test["package"], "-run", testName))
 
+def parse_source(remote):
+    if remote.lower().strip("/") == 'https://github.com/ethereum/go-ethereum':
+        return 'foundation'
+    elif remote.lower().strip("/") == 'https://github.com/maticnetwork/bor/':
```

Comment:
Stripping "/" means we need to remove that from the end of what we're checking against.